uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/provider/InfinispanContext.java

```diff
@@ -58,6 +60,8 @@
     private static final String USERNAME = "org.appformer.ext.metadata.infinispan.username";
     private static final String PASSWORD = "org.appformer.ext.metadata.infinispan.password";
     private static final String REALM = "org.appformer.ext.metadata.infinispan.realm";
+    private static final String SERVER_NAME = "org.appformer.ext.metadata.infinispan.server.name";
+    private static final String SASL_QOP = "org.appformer.ext.metadata.infinispan.sasl.qop";
```

Comment:
The properties should be added to the documentation.

Reply:
I sent an email to Docs to see where can I write this into docs, as soon they answer I will create a new PR for that (is in my task list)